python/shark_turbine/dynamo/importer.py

```diff
@@ -373,6 +373,18 @@ def _import_torch_op_overload(
         if schema.overload_name != "":
             mlir_op_name += f".{schema.overload_name}"
 
+        # Aot_autograd is not returning scalar versions of the below ops, when the second argument is scalar.
+        # Thus, intervening to select the right ops.
+        if mlir_op_name in ["torch.aten.mul.Tensor", "torch.aten.div.Tensor", "torch.aten.add.Tensor",
```

Comment:
Can you bust this list out as a module level constant and an appropriate comment?

(when you do that, might as well store it as a `set()` for efficiency of lookup)

Reply:
Actually, I would probably make this a dict so that I could map the original op name to the revised op name and avoid the special case rewriting below (i.e. just do a `get()` on the dict and use the substition if not None).